### DISCUSSION TITLE
React Bubble & Floating Menus - Tree shakeable?

### DISCUSSION THREAD

**Discussion by gonzofish** [ueberdosis/tiptap#1598] [General Questions]
I noticed that `@tiptap/react` [has `@tiptap/extension-bubble-menu` & `@tiptap/extension-floating-menu` as dependencies](https://github.com/ueberdosis/tiptap/blob/main/packages/react/package.json#L29-L30). Are they tree shakeable if you don't use them? 

Does anyone know why they weren't made `peerDependencies`?

**Comment by hanspagel** [comment]
Yes
